test/onemkl.jl

```diff
@@ -62,8 +62,17 @@ m = 20
             @test Array(dy) == x
         end
 
+        @testset "dot" begin
+            @test testf(dot, rand(T,m), rand(T,m))
+            if T == ComplexF32 || T == ComplexF64
+                @test testf(oneMKL.dotu, m, 
+                            oneArray(rand(T,m)),
+                            oneArray(rand(T,m)))
+            end
+
```

Comment:
'end' is missed??

Reply:
Good catch, yes looks like a merge issue.